src/program.js

```diff
@@ -442,6 +442,7 @@ Example: $0 --help run.
         default: 'firefox-desktop',
         demandOption: false,
         type: 'array',
+        choices: ['firefox-desktop', 'firefox-android'],
```

Comment:
Adding `choices` here automatically causes them to be added to the help test, when `web-ext run --help` is used. Now the `description` (a few lines back) can also be simplified. Could you remove the text between parentheses to shorten the description?